x11/pango/Portfile

```diff
@@ -45,13 +45,14 @@ depends_lib \
     port:fribidi \
     port:harfbuzz
 
-patchfiles-append       fix-pango_layout_get_pixel_extents-crash.patch
+configure.args-append   -Dxft=disabled \
+                        -Dintrospection=enabled
 
-configure.args          --enable-static \
-                        --disable-silent-rules \
-                        --without-xft
+configure.cflags-append \
```

Comment:
Maybe add a not about why this is required, since this isn’t needed for newer versions of glib